backup-parameter-store/riff-raff.yaml

```diff
@@ -0,0 +1,25 @@
+stacks: [frontend]
+regions: [eu-west-1]
+
+deployments:
+  backup-parameter-store:
+    type: aws-lambda
+    parameters:
+      bucket: aws-frontend-artifacts
+      fileName: backup-parameter-store.jar
+      functions:
+        PROD:
+          name: frontend-PROD-backup-parameter-store-Lambda-7XXT59MIFU7D
+          filename: backup-parameter-store.jar
+        CODE:
+          name: frontend-CODE-backup-parameter-store-Lambda-FBLJ3W8PGXM9
+          filename: backup-parameter-store.jar
+
+  backup-parameter-store-cfn:
+    type: cloud-formation
+    app: backup-parameter-store
+    parameters:
+      prependStackToCloudFormationStackName: false
+      cloudFormationStackName: backup-parameter-store
+      templatePath: cfn.yaml
+    dependencies: [backup-parameter-store]
```

Comment:
how can the lambda cloudformation depends on the lambda itself?

Reply:
It's catch 22. I've done it like this since backup-parameter-store-cfn relies on the artifact being uploaded to S3, which is what backup-parameter-store does

Reply:
While backup-parameter-store relies on the lambda being created by backup-parameter-store-cfn!
Do you know how to do this properly?